views/index.html

```diff
@@ -19,11 +19,36 @@
   <body>
     <div class="container">
       <div class="row">
-        <div class="col-md-8 col-md-offset-2">
-          <h1>Under Construction</h1>
-          <h4><a href="/api">Read My API Documentation</a></h4>
+        <div class="col-sm-12">
+          <h1>Click on an icon below</h1>
         </div>
       </div>
+      <div class="row nav-styles">
+        <div class="col-sm-3"></div>
+        <div class="col-sm-2 text-center">
```

Comment:
Again, having these 3 icons where 2 of them did nothing is a confusing structure for your site.